2023/09.md

```diff
@@ -10,6 +10,17 @@
 - **Location**: TOKYO, JAPAN
 - **Attendee information**: https://github.com/tc39/Reflector/issues/499
 
+```mermaid
+gantt
+    dateFormat  YYYY/MM/DD HH:mm Z
+    1 month before the meeting: milestone, 2023/08/26 08:00 +0900,1m
```

Comment:
would the chart be smaller and more readable if it was like, a day before the deadline?

Reply:
then todayMarker won't show 1 day before the deadline...

Reply:
ahhh ok, so it's not until it passes that date that the marker shows up, got it